ledger-service/http-json/src/it/scala/http/WebsocketServiceIntegrationTest.scala

```diff
@@ -408,7 +409,7 @@ class WebsocketServiceIntegrationTest
           via parseResp runWith remainingDeltas)
       } yield
         inside(rescan) {
-          case (Vector((fstId, fst), (sndId, snd)), Vector(observeConsumed), Some(_)) =>
+          case (Vector((fstId, fst @ _), (sndId, snd @ _)), Vector(observeConsumed), Some(_)) =>
```

Comment:
what is the differnces `fst @ _` vs `fst`. I don't see `fst` used in the scope of this pattern match, can it be be:
```
case (Vector((fstId, _), (sndId, _)), Vector(observeConsumed), Some(_))
```